Automattic/woocommerce-payments
# Reporting: Add Empty state screen UI on Payments Activity widget

Fixes #8557 

#### Changes proposed in this Pull Request

Add a `No payments yet!` view on the Payment widget, when the merchant's lifetime transactions on WooPayments is zero. 

The PR checks for `lifetimeTPV` value of an account expose through `wcpaySettings`. The value of `lifetime_total_payments_volume` is exposed via server through a https://github.com/Automattic/woocommerce-payments-server/pull/4775. 

#### Testing instructions
* On client, check out this branch. Make sure you build all static assets with `npm start`
* Browse to `Payments`. If your connected account has at least one payment transaction, you should see the payments activity widget with only its header, as seen in screenshot below: 

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/4a9ed039-77cc-4469-a2ed-75aaeaf91061)

* Connect to a fresh WooPayments account with no transactions yet, or force `hasAtLeastOnePayment` to `false` [here](https://github.com/Automattic/woocommerce-payments/blob/f2068cb3e860b7ecb95320971d93af435465032b/client/components/payment-activity/index.tsx#L18). 

You should see the empty state screen as per screenshot below:

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/424056b5-3ca4-4668-9e93-5003150b81ff)


-------------------


**Post merge**
